examples/js/renderers/SoftwareRenderer.js

```diff
@@ -713,7 +713,8 @@ THREE.SoftwareRenderer = function ( parameters ) {
 				mpUV12 = new THREE.Vector2(),
 				mpUV23 = new THREE.Vector2(),
 				mpUV31 = new THREE.Vector2(),
-				tempFace = { vertexNormalsModel : [] };
+				tempFace = { vertexNormalsModel : []
+							, color : { r: face.color.r, g: face.color.b, b: face.color.b } };
```

Comment:
in fact, a bunch of new THREE.Vector-s above could also be done once somewhere outside, probably...